### PR TITLE
Add universal redirect example

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

**Comment by sedubois** [comment]
Would be nice to have a deployment on now to see how it behaves 😊

**Comment by timneutkens** [comment]
@sedubois great idea, done 🙌

File: examples/with-universal-redirect/pages/_error.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ErrorComponent from 'next/error'
+import redirect from 'next-universal-redirect'
```
Comment (arunoda) [inline examples/with-universal-redirect/pages/_error.js]: Shall we make this open? I can't see the source.
See: https://github.com/timneutkens/next-redirect

**Comment by sedubois** [comment]
What about https://github.com/matthewmueller/next-redirect by @matthewmueller?

**Comment by sergiodxa** [comment]
I think `next-redirect` is easier to use than `next-universal-redirect`, is just a function you call in `getInitialProps`, you don't need a HOC and create a Map instance, just call a simple function 🤔

File: examples/with-universal-redirect/pages/_error.js (hunk 1 above)
Comment (timneutkens) [inline examples/with-universal-redirect/pages/_error.js]: https://github.com/timneutkens/next-universal-redirect

**Comment by timneutkens** [comment]
@sergiodxa it serves a different purpose. This one is for lists of redirects from non-existent pages to existent (well, if you screw up non-existent) pages. Next-redirect is for a more for redirecting when stuff happens. Also, next-redirect is not universal but only server side.

File: examples/with-universal-redirect/pages/_error.js (hunk 1 above)
Comment (timneutkens) [inline examples/with-universal-redirect/pages/_error.js]: Fixed 🙌

**Comment by matthewmueller** [comment]
> next-redirect is not universal but only server side.

actually it works on both: https://github.com/matthewmueller/next-redirect/blob/master/index.js#L16

**Comment by timneutkens** [comment]
@matthewmueller ah, right, I meant it doesn't use `next/router` 😄

**Pull request closed without merging**